### PR TITLE
two new church windows

### PR DESCRIPTION
**Pull request by bobjension**
windows are good

### CODE REVIEW

**Comment by AMyriad** [comment]
image
bro

File: code/game/objects/structures/roguewindow.dm [hunk 1]
```diff
@@ -32,12 +32,24 @@
 	icon_state = "[base_state]"
 
 /obj/structure/roguewindow/stained
-	icon_state = "stained-silver"
-	base_state = "stained-silver"
+	icon_state = null
```
Comment (AMyriad) [inline code/game/objects/structures/roguewindow.dm:35]: just don't include `icon_state` and `base_state` variables here if this isn't going to have any

Comment (bobjension) [reply code/game/objects/structures/roguewindow.dm:35]: nah

**Comment by bobjension** [comment]
> image bro

fixed the vamp stuff

**Comment by bobjension** [comment]
forgot about the fucking loom PR, now IT should be good and ready to merged

**Comment by Real-MAGNUM** [comment]
Testmerging

**Comment by Real-MAGNUM** [comment]
Sure seems fine to me.

**Pull request merged**